.github/workflows/auto_update_libs.yaml

```diff
@@ -0,0 +1,10 @@
+name: Auto-update charm libraries
```

Comment:
I've just realized something: do you think it would be useful to add a badge for this workflow?

Reply:
Not sure if it makes sense. What do you think?

Reply:
Anyway, I'd say that's out of the scope of this PR